src/pages/GamePage.tsx

```diff
@@ -43,6 +47,7 @@ export default function GamePage() {
               );
             }
           })}
+          {amMaster && <EmptyPlayerPLace />}

```

Comment:
Please correct the name of component, i.e. the word: PLace -> Place

Reply:
Good point, i'm on it